Aliases for non-source and falling water and lava

sjoerdtim opened the pull request:
Aliases for each of the stages of non-source flowing or falling water blocks.

TheBentoBox commented:
Looks good, but this would need to be split out so that you're only using block states in 1.13 or newer. You can do that with a `minecraft version` tag at the top of the section like we do with most alias sections.

sjoerdtim commented:
@TheBentoBox Thanks. I have added the version tags. Can you confirm if I did those correctly?

TheBentoBox commented:
The one thing I notice is that now there's no lava alias in the `1.13 or newer` section so lava would be unaliased in the newer versions.

Also, only just noticed, but your flowing stages should be using a vertical pipe `|` instead of `/` to separate the options within parentheses. So for example instead of `flowing stage (1/one)` you'd do `flowing stage (1|one)`.

Otherwise, looks good and it's a good change 👍

sjoerdtim commented:
@TheBentoBox Thanks! I have fixed these issues and also added the flowing states for lava.
Sorry for the mistakes in the beginning, this is my first pull request for Skript ever, so I needed to get myself familiar with the syntaxes here first!

TheBentoBox commented:
No worries; thanks for the contribution! 😄
